frontend/src/components/samples/SampleTableColumns.tsx

```diff
@@ -103,6 +105,55 @@ export const SAMPLE_COLUMN_DEFINITIONS: { [key in SampleColumnID]: SampleColumn
 		},
 	},
 
+	[SampleColumnID.PARENT_CONTAINER]: {
+		columnID: SampleColumnID.PARENT_CONTAINER,
+		title: 'Parent Container',
+		dataIndex: ['sample', 'container'],
+		render: (_, { sample }) => {
+			return (
+				sample &&
+				sample.container && (
+					<WithContainerRenderComponent
+						objectID={sample.container}
+						placeholder={<span>loading...</span>}
+						render={(container) =>
+							container.location ?
+								<WithContainerRenderComponent
+									objectID={container.location}
+									placeholder={<span>loading...</span>}
+									render={(container) => <Link to={`/containers/${container.id}`}>{container.barcode}</Link>}
+								/>
+								: <></>}
+					/>
+				)
+			)
+		},
+	},
+	
+	[SampleColumnID.PARENT_COORDINATES]: {
+		columnID: SampleColumnID.PARENT_COORDINATES,
+		title: 'Parent Coordinates',
```

Comment:
you can use "Parent Coords" to mirror the other column and prevent taking space for nothing.